app/uk/gov/hmrc/thirdpartyapplication/connector/EmailConnector.scala

```diff
@@ -56,26 +56,28 @@ class EmailConnector @Inject()(httpClient: HttpClient, config: EmailConfig)(impl
   val addedClientSecretNotification = "apiAddedClientSecretNotification"
   val removedClientSecretNotification = "apiRemovedClientSecretNotification"
 
-  def sendAddedCollaboratorConfirmation(role: String, application: String, recipients: Set[String])(implicit hc: HeaderCarrier): Future[HttpResponse] = {
+  def sendAddedCollaboratorConfirmation(role: String, application: String, recipients: Set[String])(implicit hc: HeaderCarrier): Future[Unit] = {
```

Comment:
^We definitely need to do more of this - shouldn't be leaking `HttpResponse` outside of the connector